explainaboard/processors/named_entity_recognition.py

```diff
@@ -598,6 +598,11 @@ def get_bucket_performance_ner(
                 samples_over_bucket_pred,
             )
 
+            bucket_performance = BucketPerformance(
+                bucket_name=bucket_interval,
+                n_samples=len(spans_pred),
+                bucket_samples=bucket_samples,
+            )
             for metric_name in sys_info.metric_names:
                 """
                 # Note that: for NER task, the bucket-wise evaluation function is a little different from overall
```

Comment:
Yeah, and the way to calculate the confidence interval is different as well, relatively complicated, I think.